<launch>

    <arg name="solver" default="OpenSot"/>
    <arg name="prefix" default=""/>
    <param name="robot_description" textfile="$(find cogimon_urdf)/urdf/cogimon.urdf"/>
    <param name="robot_description_semantic" textfile="$(find cogimon_srdf)/srdf/cogimon.srdf"/>
    <param name="problem_description" textfile="$(find cartesian_interface)/examples/configs/cogimon_simple_stack.yaml"/>
    

    <node pkg="cartesian_interface" type="ros_server_node" name="ros_server_node" required="true" output="screen" launch-prefix="$(arg prefix)">
	<param name="is_model_floating_base" type="bool" value="true"/>
        <param name="model_type" value="RBDL"/>
        <param name="solver" value="$(arg solver)"/>
        <param name="world_frame" value="l_sole"/>
    </node>

    <node pkg="cartesian_interface" type="marker_spawner" name="interactive_markers" output="screen"/>

</launch>
